README.txt
This extension is a FE plugin that manages relations between Tesseract components and produces output in the FE.

Refer to the manual for more information.

ChangeLog
2013-11-06 Francois Suter (Cobweb)  <typo3@cobweb.ch>

	* Removed useless require_once() call, resolves #53387

2013-06-21 Francois Suter (Cobweb)  <typo3@cobweb.ch>

	* Released to TER as version 1.5.0

2013-06-14 Francois Suter (Cobweb)  <typo3@cobweb.ch>

	* Improved frontend debug output, removed Kint dependency, resolves #49120

2013-05-22 Francois Suter (Cobweb)  <typo3@cobweb.ch>

	* Verified compatibility with TYPO3 CMS 6.1, resolves #47457
	* Added note about preview mechanisms incompatibilities, resolves #48431

2013-04-11 Francois Suter (Cobweb)  <typo3@cobweb.ch>

	* Verified compatibility with TYPO3 CMS 6.0, resolves #47122

2013-04-09 Francois Suter (Cobweb)  <typo3@cobweb.ch>

	* Added flexform option for local debugging, resolves #33212

2012-09-03 Francois Suter (Cobweb)  <typo3@cobweb.ch>

	* Released to TER as version 1.4.1

2012-08-20 Roberto Presedo (Cobweb)  <typo3@cobweb.ch>

	* Fixed bug with hidden records while previewing from the BE, resolves #39991

2012-07-10 Francois Suter (Cobweb)  <typo3@cobweb.ch>

	* Released to TER as version 1.4.0

2012-06-24 Francois Suter (Cobweb)  <typo3@cobweb.ch>

	* Switched to the sprite manager for plugin icons, resolves #38304

2012-06-22 Francois Suter (Cobweb)  <typo3@cobweb.ch>

	* Verified compatibility with TYPO3 4.7, raised requirement to TYPO3 4.5, resolves #38291
	* Removed some pre-4.5 code, corrected phpDocs, removed obsolete code

2012-04-19 Francois Suter (Cobweb)  <typo3@cobweb.ch>

	* Released to TER as version 1.3.0

2012-03-20 Francois Suter (Cobweb)  <typo3@cobweb.ch>

	* Added possibility to configure preview for "Save and view" action, resolves #34594
	* Changed code to delegate creation of empty data structure to providers, resolves #10980

2012-02-06 Francois Suter (Cobweb)  <typo3@cobweb.ch>

	* Released to TER as version 1.2.0

2012-01-30 Francois Suter (Cobweb)  <typo3@cobweb.ch>

	* Implemented partial refactoring of tx_displaycontroller::main(), resolves #33527
	* Improved check for debugger class existence, resolves #33528
	* Added cloning of secondary provider to avoid prodivers overriding each other, resolves #33529

2012-01-21 Francois Suter (Cobweb)  <typo3@cobweb.ch>

	* Moved debug output to a separate class, made it configurable, resolves #33360

2012-01-16 Francois Suter (Cobweb)  <typo3@cobweb.ch>

	* Removed all mentions of contexts from the manual, updated the hooks list, resolves #33223
	* Added Kint to dump debug data instead of t3lib_utility_Debug::debug(), resolves #33174

2012-01-13 Francois Suter (Cobweb)  <typo3@cobweb.ch>

	* Enhanced error reporting, resolves #1695
	* Implemented new base class for pi-based controllers, resolves #33187

2012-01-12 Francois Suter (Cobweb)  <typo3@cobweb.ch>

	* Implemented new message queue API, resolves #33149

2012-01-08 Francois Suter (Cobweb)  <typo3@cobweb.ch>

	* Implemented initialization of components using the new Tesseract API, resolves #33050

2011-12-28 Francois Suter (Cobweb)  <typo3@cobweb.ch>

	* Changed compatibility check between Primary Data Provider and Data Consumer, resolves #32863

2011-08-16 Francois Suter (Cobweb)  <typo3@cobweb.ch>

	* Released to TER as version 1.1.0

2011-07-29 Francois Suter (Cobweb)  <typo3@cobweb.ch>

	* Adapted layout of content element fields to match TYPO3 4.5 UI, resolves #28572

2011-07-18 Francois Suter (Cobweb)  <typo3@cobweb.ch>

	* Changed empty filter check to use new API, resolves #28288

2011-05-18 Francois Suter (Cobweb)  <typo3@cobweb.ch>

	* Released to TER as version 1.0.2

2011-05-16 Francois Suter (Cobweb)  <typo3@cobweb.ch>

	* Corrected handling of components relations in workspaces, resolves #26828

2011-03-20 Francois Suter (Cobweb)  <typo3@cobweb.ch>

	* Removed dependency on extensions load order for allowed property, resolves #13929

2010-12-28 Francois Suter (Cobweb)  <typo3@cobweb.ch>

	* Released to TER as version 1.0.1

2010-12-20 Francois Suter (Cobweb)  <typo3@cobweb.ch>

	* Changed consumer member variable from static to "normal" in the displaycontroller workflow, resolves #11390 (thanks to Adrien Crivelli)
	* Removed tx_displaycontroller::sub() method, resolves #11624

2010-12-20 Francois Suter (Cobweb)  <typo3@cobweb.ch>

	* Added FAQ in manual, with first question about trouble when saving a displaycontroller, references #11549

2010-10-23 Francois Suter (Cobweb)  <typo3@cobweb.ch>

	* Corrected merging of TypoScript configuration, resolves #10410

2010-09-27 Francois Suter (Cobweb)  <typo3@cobweb.ch>

	* First public release

ext_emconf.php
<?php

/*********************************************************************
* Extension configuration file for ext "displaycontroller".
*
* Generated by ext 21-06-2013 13:05 UTC
*
* https://github.com/t3elmar/Ext
*********************************************************************/

$EM_CONF[$_EXTKEY] = array (
  'title' => 'Standard Controller - Tesseract project',
  'description' => 'This FE plugin manages relations between Tesseract components and produces output in the FE. More info on http://www.typo3-tesseract.com/',
  'category' => 'plugin',
  'author' => 'Francois Suter (Cobweb)',
  'author_email' => 'typo3@cobweb.ch',
  'shy' => '',
  'dependencies' => 'cms,tesseract',
  'conflicts' => '',
  'priority' => '',
  'module' => '',
  'state' => 'stable',
  'internal' => '',
  'uploadfolder' => 0,
  'createDirs' => '',
  'modify_tables' => '',
  'clearCacheOnLoad' => 0,
  'lockType' => '',
  'author_company' => '',
  'version' => '1.5.0',
  'constraints' => 
  array (
    'depends' => 
    array (
      'cms' => '',
      'typo3' => '4.5.0-6.1.99',
      'tesseract' => '1.4.0-0.0.0',
    ),
    'conflicts' => 
    array (
    ),
    'suggests' => 
    array (
    ),
  ),
  'suggests' => 
  array (
  ),
  '_md5_values_when_last_written' => 'a:31:{s:9:"ChangeLog";s:4:"1a27";s:30:"class.tx_displaycontroller.php";s:4:"dbcc";s:39:"class.tx_displaycontroller_debugger.php";s:4:"38b7";s:38:"class.tx_displaycontroller_realurl.php";s:4:"2698";s:38:"class.tx_displaycontroller_service.php";s:4:"8edb";s:30:"displaycontroller_typeicon.png";s:4:"103f";s:16:"ext_autoload.php";s:4:"58ed";s:21:"ext_conf_template.txt";s:4:"e283";s:12:"ext_icon.gif";s:4:"f02f";s:17:"ext_localconf.php";s:4:"d4f2";s:14:"ext_tables.php";s:4:"a570";s:14:"ext_tables.sql";s:4:"62a7";s:13:"locallang.xml";s:4:"eef3";s:27:"locallang_csh_ttcontent.xml";s:4:"a95d";s:16:"locallang_db.xml";s:4:"bc5c";s:10:"README.txt";s:4:"b948";s:15:"wizard_icon.gif";s:4:"b025";s:14:"doc/manual.pdf";s:4:"f7a5";s:14:"doc/manual.sxw";s:4:"56d1";s:14:"doc/manual.txt";s:4:"1bc2";s:50:"hooks/class.tx_displaycontroller_hooks_tcemain.php";s:4:"44e3";s:27:"lib/kint/config.default.php";s:4:"ae73";s:23:"lib/kint/Kint.class.php";s:4:"dcea";s:22:"lib/kint/view/kint.css";s:4:"899b";s:21:"lib/kint/view/kint.js";s:4:"f06a";s:25:"lib/kint/view/trace.phtml";s:4:"facf";s:38:"pi1/class.tx_displaycontroller_pi1.php";s:4:"3994";s:46:"pi1/class.tx_displaycontroller_pi1_wizicon.php";s:4:"614d";s:38:"pi2/class.tx_displaycontroller_pi2.php";s:4:"9ddb";s:46:"pi2/class.tx_displaycontroller_pi2_wizicon.php";s:4:"a48d";s:16:"static/setup.txt";s:4:"b445";}',
  'user' => 'francois',
  'comment' => 'Compatibility with TYPO3 6.0 and 6.1; improved debugging.',
);

?>